Dockerfile

```diff
@@ -11,8 +11,12 @@ RUN apt-get update && \
 ARG mirrorbits_version=v0.5.1
 RUN mkdir /mirrorbits && \
   curl -L https://github.com/etix/mirrorbits/releases/download/${mirrorbits_version}/mirrorbits-${mirrorbits_version}.tar.gz -O && \
-  tar xvzf /mirrorbits-${mirrorbits_version}.tar.gz -C / && \
-  rm /mirrorbits-${mirrorbits_version}.tar.gz
```

Comment:
Removed as this layer is not kept in the final image, no need for cleaning it up.